charts/spire/templates/_spire-lib.tpl

```diff
@@ -32,9 +32,11 @@
 
 {{- define "spire-lib.registry" }}
 {{- if ne (len (dig "spire" "image" "registry" "" .global)) 0 }}
-{{- .global.spire.image.registry }}
+{{- print .global.spire.image.registry "/"}}
+{{- else if ne (len (.image.registry)) 0 }}
+{{- print .image.registry "/"}}
 {{- else }}
-{{- .image.registry }}
+{{- print ""}}
```

Comment:
Can leave out the else and print.

Reply:
how would that work? Need the else for the empty registry field case.

Reply:
the macro's just printing stuff out. you have logic to print nothing out, but actually not doing anything also has the same affect? Basically the else and print are doing nothing but taking cycles?

Reply:
Got it, thanks for clarifying

Reply:
Is this giving an image like `/spire-server` is that actually a valid image name starting with a `/` haven't seen that before.

Reply:
Without the change it would have an extra / in front. Depending on the container runtime, that may or may not work. Dropping out the / in the case where registry is overridden to be "" seems reasonable to me.

Reply:
What Kevin said. Without this change docker complains about the extra slash in front. 

Removed the extra else statement.